addons/frag/functions/fnc_dev_trackObj.sqf

```diff
@@ -0,0 +1,104 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * This function adds an object to have it's course tracked (every frame).
+ *
+ * Arguments:
+ * 0: Object to draw hitbox <OBJECT>
+ * 1: Color of trace <STRING>
+ * 2: Whether the object is a projectile <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call ace_frag_fnc_dev_trackObj;
+ *
+ * Public: No
+ */
+params [
+    "_obj",
+    ["_color", "blue", ["blue"]],
+    ["_isProj", false, [false]]
+];
+TRACE_4("devDraw",_this,_obj,_color,_isProj);
+
+// pick color and add it to the array
+private _colorArray = switch (toLower _color) do {
```

Comment:
```suggestion
private _colorArray = switch (toLowerANSI _color) do {
```

Reply:
Addressed in https://github.com/acemod/ACE3/pull/9728/commits/3f45f7735b6b6c50f01d6fcb50db08f20548c0c4